docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+license_link: {license_link}                            # Required. if license = other, specify "LICENSE" or "LICENSE.md" to link to a file of that name inside the repo, or a URL to a remote file.
+library_name: {library_name}                            # Optional. Example: keras or any library from https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/model-libraries.ts.
+tags:
+- {tag_0}                                               # Optional. Example: audio
+- {tag_1}                                               # Optional. Example: automatic-speech-recognition
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+model-index:
+- name: {model_id}                                      # Required. Example: CatClassifier.
+  model: {model_uri}                                    # Required. URI to a repository containing the model file.
+  results:
+  - task:
+      type: {task_type}                                 # Required. Example: image-classification.
+      name: {task_name}                                 # Optional. Example: Image Classification.
+    dataset:
+      type: {dataset_type}                              # Required. Example: common_voice. Link to a repository containing the dataset or a dataset id from https://hf.co/datasets.
+      name: {dataset_name}                              # Required. Example: "Common Voice (French)". A pretty name for the dataset.
+      revision: {dataset_version}                       # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
+    metrics:
+    - type: {metric_type}                               # Required. Example: false-positive-rate. Use metric id from https://hf.co/metrics.
+      name: {metric_name}                               # Required. Example: "FPR wrt class 0 restricted to feature gender:0 and age:21".
+      value: {metric_value}                             # Required. Example: 0.75.
+      class: {metric_output_class}                      # Optional. Example: "1". Only relevant for certain metrics such as for example false-positive-rate positive rate.
+      subgroups:
+        - feature: {feature_name}                       # Optional. Example: "gender".
+          group: {feature_subgroup}                     # Optional. Example: "female".
+    measurements:
+      # Bar plots should be able to capture SHAP and Robustness Toolbox from AI Verify.
```

Comment:
Is SHAP a logical example? Because SHAP is used to explain the output of a machine learning model. I.s.o. the "model card" I would expect to have such an instrument listed for something like a "output card". What do you think?

Reply:
I'm not sure I understand this comment. What is an "output card"?

Reply:
SHAP can be used to explain how important features are on for example train or test data (as far as my understanding goes). My assumption was that we would like to capture these kind of test results in our model card.  At this point I would just stick to one thing, which we can call a "model card", and in the future we will probably split the fields among different "cards".